src/user-post/service/post.service.ts

```diff
@@ -0,0 +1,188 @@
+import { Injectable } from '@nestjs/common';
+import { Posts } from '../entities/Post.entity';
+import { EditPostDto, PostDto } from '../dto/post.dto/post.dto';
```

Comment:
Why we have so many `post.dto` on the same path?

Reply:
now they are separated